pysb/core.py

```diff
@@ -362,7 +362,7 @@ def __init__(self, monomer, site_conditions, compartment):
         invalid_sites = []
         for (site, state) in site_conditions.items():
             # pass through to next iteration if state type is ok
-            if state == None:
+            if state is None and site not in monomer.site_states.keys():
```

Comment:
Written more simply as `... not in monomer.site_states`.

Reply:
Fixed